.prettierrc.json

```diff
@@ -17,11 +17,14 @@
   "useTabs": false,
   "overrides": [
     {
-      "files": ["docs/typescript/**/*.md"],
+      "files": [
+        "docs/typescript/**/*.md",
+        "docs-src/typescript/*.md"
```

Comment:
I thought it hit the whole repo by default...
Why wouldn't we want prettier to run on everything in docs-src ?

Reply:
This doesn't change the script (`"format": "prettier --write .",`)—these are just the file paths for the TS prettier rules that are different from the default/general rules.

Reply:
It isn't that we don't want to run prettier on `docs-src/typescript/**/*.md`, we just want to apply a couple of rule overrides to those directories.